qiita_plugins/target_gene_type/target_gene_type/summary.py

```diff
@@ -0,0 +1,173 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+import matplotlib.pyplot as plt
```

Comment:
can you put this import below the one from `qiita_ware.demux`? (pep8 import order)